TypeCobol.Analysis.Test/Report/InBulkCallPgm88Set.csv

```diff
@@ -1,7 +1,7 @@
 ﻿ObjectName=PGM00000;Line=16;Column=12;Path=ProcCall.PROGRAM_NAME<-"PGM00000";SourceText=call 'zcallpgm' using PROGRAM_NAME PARAMETER_1 PARAMETER_2
-ObjectName=PGM00001;Line=21;Column=12;Path=ProcCall.PROGRAM_NAME.pgm1<-"true";SourceText=call 'zCaLlpgm' using by reference PROGRAM_NAME
-ObjectName=PGM00002;Line=24;Column=12;Path=ProcCall.PROGRAM_NAME.pgm2<-"true";SourceText=call 'zCaLlpgF' using PROGRAM_NAME
-ObjectName=PGM00003;Line=28;Column=12;Path=ProcCall.PROGRAM_NAME.pgm3<-"true";SourceText=call 'zcallpgr' using PROGRAM_NAME PARAMETER_1 PARAMETER_2
+ObjectName=PGM00001;Line=21;Column=12;Path=ProcCall.PROGRAM_NAME<-ProcCall.PROGRAM_NAME.pgm1<-"PGM00001";SourceText=call 'zCaLlpgm' using by reference PROGRAM_NAME
```

Comment:
This is a point of view, I would like to point out that it was because of setting to true the variable condition, but this means that the main variable takes the condition variable value.

Reply:
Well it's still doable, the `ValueOrigin`  class does not deal with the representation of the values and variables used in a path. We just have to agree on a format and i'll change the `AsPaths` method accordingly.
What about something like `ProcCall.PROGRAM_NAME<-"PGM00001"<-ProcCall.PROGRAM_NAME.pgm1<-"true"` ?

Reply:
It's ok

Reply:
Modified